sympy/holonomic/holonomic.py

```diff
@@ -1215,9 +1215,17 @@ def _frobenius(self, lb=True):
         indicialroots = self._indicial()
 
         reals = []
+        compl = []
         for i in indicialroots:
             if i.is_real:
                 reals.extend([i] * indicialroots[i])
+            else:
+                a, b = i.as_real_imag()
+                compl.extend([(i, a, b)] * indicialroots[i])
+        compl.sort(key=lambda x : x[1])
+        for i in range(0, len(compl)-1):
+            if compl[i][2] > 0 and compl[i][1] == compl[i+1][1]:
```

Comment:
There may be several roots having the same real part. If the coefficients of the indicial equation are not real, the roots are not necessarily conjugates of each other.